### PR TITLE
Draft for Installation Page

### PR DESCRIPTION
**Pull request by Hanawa02**
- Creates a draft for Installation Page content

### CODE REVIEW

File: src/pages/installation.md [hunk 1]
```diff
@@ -2,4 +2,70 @@
...
+npm install astro
+```
+
+or
+
+```bash
+yarn add astro
+```
+
+And now you have astro installed and can start implementing your application.
```
Comment (itskitto) [inline src/pages/installation.md:71]: maybe "building" instead of "implementing"?

**Review by itskitto — CHANGES_REQUESTED** [review]
Just a a few suggestions to improve clarity during reading

File: src/pages/installation.md [hunk 2]
```diff
@@ -2,4 +2,70 @@
...
+
+### Node.js
+
+To use astro you will need to have Node.js installed in your machine. If you don't have it yet, you can download it in the [official site](https://nodejs.org/).
+
+> Important: Astro is built with ESM modules which are not supported in older version of Node.js. The minimum supported version is 14.15.1.
+
+### Package Manager
+
+You also need a package manager to use astro, if you already installed `Node.js` you already have [npm](https://docs.npmjs.com/about-npm).
```
[the remark this answers is not in the document]
Comment (Hanawa02) [reply src/pages/installation.md]: Fixed the capitalization, also for other places that weren't code snippets.

File: src/pages/installation.md (hunk 1 above)
Comment (Hanawa02) [reply src/pages/installation.md]: I was unsure about which word would fit better. I believe building sounds better than implementing, so I changed already. Our writing experts will probably improve again later on.

File: src/pages/installation.md [hunk 3]
```diff
@@ -2,4 +2,70 @@
...
 
-# Installation
\ No newline at end of file
+# Installation
+
+## Dependencies
+
+### Node.js
+
+To use astro you will need to have Node.js installed in your machine. If you don't have it yet, you can download it in the [official site](https://nodejs.org/).
```
[the remark this answers is not in the document]
Comment (Hanawa02) [reply src/pages/installation.md]: I changed to your suggestion, my grammar/spelling can be quite off so PLEASE point everything you see wrong or not fitting, I will appreciate it.

**Comment by itskitto** [comment]
I'm good to go on this one!

**Review by itskitto — APPROVED** [review]

**Comment by FredKSchott** [comment]
Looks great!

**Pull request merged**